test/regridder_tests.jl

```diff
@@ -2,6 +2,7 @@
     Unit tests for ClimaCoupler Regridder module
 =#
 
+using ClimaCoupler
```

Comment:
we might need it to be able to call `pkgdir(ClimaCoupler)`, but it would be nice to use `import` instead of `using` to follow our new code loading convention